internal/controllers/clusterextension_controller.go

```diff
@@ -206,6 +207,11 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1.Cl
 	installedBundle, err := r.InstalledBundleGetter.GetInstalledBundle(ctx, ext)
 	if err != nil {
 		setInstallStatus(ext, nil)
+		var saerr *authentication.ServiceAccountNotFoundError
+		if errors.As(err, &saerr) {
+			l.Info("ServiceAccount defined in the ClusterExtension is not found")
+			err = saerr
+		}
```

Comment:
By returning this error in line 217, aren't we getting logging by controller-runtime already? If so, this seems extraneous.

Reply:
removing the log